iam/role/README.md

```diff
@@ -0,0 +1,80 @@
+# IAM Role Service Acorn
+
+This Service Acorn creates a CloudFormation stack containing the given IAM Role and inline Policy.
+
+## Limitations
+
+Currently, this Service Acorn only supports trust relationships with a single ARN.
+The ARN can refer to an AWS account, an IAM user, an IAM role, or specific assumed-role sessions.
+
+## Usage
+
+### Running the Acorn
+
+```
+acorn run ghcr.io/acorn-io/aws/iam/role:v0.1.0 \
+  --role-name="my-role" \
+  --policy @policy.json \
+  --trusted-arn="<arn>" \
+  --external-ids="external-id-one,external-id-two" \
+  --max-session-duration-minutes=120
+```
+
+### Using the service in an Acornfile
+
+```cue
+services: role: {
+    image: "ghcr.io/acorn-io/aws/iam/role:v0.1.0"
+    serviceArgs: {
+        roleName:   "my-role"
+        trustedArn: "<arn>"
+        // This is an example policy:
+        policy: {
```

Comment:
Just wanted to point out that this UX looks super clean. Great work.

Reply:
Thanks!